io.catenax.id_based_request_for_update/1.0.0/IdBasedRequestForUpdate.ttl

```diff
@@ -0,0 +1,94 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IML)
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 Mercedes Benz AG
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 SupplyOn AG
+# Copyright (c) 2023 Volkswagen AG
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.id_based_request_for_update:1.0.0#>.
+
+:weekBasedMaterialDemand a bamm:Property;
+    bamm:description "Material demands the supplier requests an update for."@en;
+    bamm:characteristic :WeekBasedMaterialDemandCharacteristic;
+    bamm:preferredName "Week Based Material Demand"@en.
+:WeekBasedMaterialDemandCharacteristic a bamm-c:List;
+    bamm:description "A list of material demand UUIDs an update is requested for."@en;
+    bamm:preferredName "Week Based Material Demand Characteristic"@en;
+    bamm:dataType :WeekBasedMaterialDemandEntity.
+:weekBasedCapacityGroupList a bamm:Property;
+    bamm:description "Capacity groups the customer requests an update for."@en;
+    bamm:characteristic :WeekBasedCapacityGroupCharacteristic;
+    bamm:preferredName "Week Based Capacity Group List"@en.
+:WeekBasedCapacityGroupCharacteristic a bamm-c:List;
+    bamm:description "A list of capacity group UUIDs an update is requested for."@en;
+    bamm:preferredName "Week Based Capacity Group Characteristic"@en;
+    bamm:dataType :WeekBasedCapacityGroupEntity.
+:WeekBasedMaterialDemandEntity a bamm:Entity;
+    bamm:description "Encapsulates the information that is necessary for an update request."@en;
+    bamm:preferredName "Week Based Material Demand Entity"@en;
+    bamm:properties (:materialDemandId :changedAt).
```

Comment:
In the new model request, changedAt was shown as optional property. Has this changed?